src/EditorFeatures/Core.Wpf/InlineRename/UI/SmartRename/SmartRenameViewModel.cs

```diff
@@ -292,6 +303,10 @@ public void ToggleOrTriggerSuggestions()
             {
                 this.FetchSuggestions(isAutomaticOnInitialization: false);
             }
+            else
+            {
+                _cancellationTokenSource?.Cancel();
+            }
```

Comment:
how can the cts ever be null?

Reply:
I instantiated it in constructor and made it not-nullable